docs/network/evm-nodes.md

```diff
@@ -59,25 +59,25 @@ You can initialize the node from a snapshot or allow it to compute the Etherlink
 
 ### From a snapshot
 
-1. Download [an Etherlink snapshot](http://snapshotter-sandbox.nomadic-labs.eu/), and use the `octez-evm-node` to import it.
+To automatically download and import a snapshot, start the node with the `--init-from-snapshot` switch, as in this example:
 
-   ```bash
-   wget http://snapshotter-sandbox.nomadic-labs.eu/etherlink-mainnet/evm-snapshot-sr1Ghq66tYK9y-latest.gz # this is for the latest mainnet etherlink snapshots, similarly there is one for testnet
-   octez-evm-node snapshot import evm-snapshot-sr1Ghq66tYK9y-latest.gz --data-dir $evm_observer_dir
+```bash
+octez-evm-node run observer --data-dir $evm_observer_dir
```

Comment:
There `init-from-snapshot` flag is not passed here.

Reply:
Just checking to see that you were paying attention. :-D I've added the flag and the --network flag, too.